DocumentFormat.OpenXml/src/Framework/OpenXmlPackage.cs

```diff
@@ -1466,7 +1516,17 @@ private string GetNextSequenceNumber(string contentType)
                 else
                 {
                     this._sequenceNumbers.Add(contentType, 1);
-                    return "";
+                    //return "";
+
+                    //Certain types need to contain an initial 1 value. Issue #123
```

Comment:
Is there a reason you're including the issue number? I haven't seen that anywhere else in the code base